src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
```

Comment:
It's considered bad practice to leave commented-out lines of code. Please make sure to remove them.